src/systems/ec/implementations/vanilla/packages/ec-component-form-checkbox/ec-component-form-checkbox.scss

```diff
@@ -0,0 +1,88 @@
+/**
+ * Checkbox component
+ * @define checkbox
+ */
+
+// Import base
+@import '@ecl/ec-base/ec-base';
+
+@mixin ecl-checkbox(
+  $thickness-base: 1px,
+  $thickness-max: 3px,
+  $border-color: $ecl-color-grey-25,
+  $border-color-focus: $ecl-color-yellow-100,
+  $border-color-checked: $ecl-color-blue-100,
+  $fake-checkbox-size: 0.875rem
+) {
+  .ecl-checkbox {
+    align-items: center;
+    display: flex;
+    position: relative;
+  }
+
+  .ecl-checkbox__input {
+    margin-left: 3px;
+    opacity: 0;
+    padding: 0;
+    position: absolute;
+    z-index: 1;
+  }
+
+  .ecl-checkbox__label {
+    color: $ecl-color-grey-100;
+    font: $ecl-font-m;
+    padding-left: $ecl-spacing-xl;
```

Comment:
to be more accurate we should have something like `padding-left: calc({box_width} + {box_padding} + {box_border} + {spacing_s}`. That way we ensure that the spacing before the label will always be exactly as on the specs.
(by chance it will give the same value, 2rem, but this is a coincidence)

Reply:
@emeryro, the padding is not anymore in the label, therefore no need to make complex calculations, i think, we are now setting the requested distance between the "checkbox" and the label  via a margin on the checkbox (the one you want to be removed when the label is hidden.. ;) )